main/acle.md

```diff
@@ -12311,11 +12311,7 @@ While (resulting in predicate-as-counter). ``vl`` is expected to be 2 or 4.
    // _c8[_u64], _c16[_u64], _c32[_u64] and _c64[_u64]
    svcount_t svwhilege_c8[_s64](int64_t rn, int64_t rm, uint64_t vl);
```

Comment:
What about other `svwhile` builtins? Are they staying the same (and inconsistent with the new one) for historical reasons?
https://developer.arm.com/architectures/instruction-sets/intrinsics/#q=svwhile